appdev/guides/importing-CH-urldispatcher.rst

```diff
@@ -89,21 +89,10 @@ And how do we manage the received URL?
 
 After the ``URLdispatcher`` sends the link to the correspondent app, we need to handle that URL or URI in the targeted app. Let’s see how to do that:
 
-In the main qml file, we need to add some code to know what to do with the dispatched URL. First add an `Arguments element <https://gitlab.com/ubports-linphone/linphone-simple/blob/master/qml/Main.qml#L189>`_ that holds the URL, as is done, for example, in the `Linphone app <https://open-store.io/app/linphone.cibersheep>`_. Also, we add `connection to the URI Handler <https://gitlab.com/ubports-linphone/linphone-simple/blob/master/qml/Main.qml#L200>`_ with a Connection element with ``UriHandler`` as a target.
+In the main qml file, we need to add some code to know what to do with the dispatched URL. Let's add `connection to the URI Handler <https://gitlab.com/ubports-linphone/linphone-simple/blob/master/qml/Main.qml#L200>`_ with a Connection element with ``UriHandler`` as a target.
```

Comment:
Let's add **a** connection

Reply:
I think it was nice to say explicitly that we are looking at one particular app (Linphone) as an example.

Reply:
- Thanks. I'll edit that.
- About Linphone, true... I wasn't sure how to proceed as the sample code should change... but first I wanted to reflect it in the docs...

Reply:
Updated :)